### PR TITLE
[JENKINS-47480] Jenkins rendering error fix in 1.3.0

### PR DESCRIPTION
**Pull request by vivek**
# Description

As reported in the ticket, port of PR #1531 fixes this issue. 

See [JENKINS-47480](https://issues.jenkins-ci.org/browse/JENKINS-47480).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by michaelneale** [comment]
woah - this :bug: build failure 🤕   is js builder/browserify/bundler. If you look at the failed build, there are 20 failures all in ATH, and the screenshots in the Artifacts tab, you see a bunch of non loading screens: 

https://ci.blueocean.io/job/blueocean/job/JENKINS-41205_large-node-handling-1.3/1/artifact/acceptance-tests/target/screenshots/io.blueocean.ath.live.creation.GitCreationTest_testHttpsPrivateRepository.png

Also note the smaller bundle size on 

https://ci.blueocean.io/blue/organizations/jenkins/blueocean/detail/JENKINS-41205_large-node-handling-1.3/1/artifacts (failed build)

vs: 
https://ci.blueocean.io/blue/organizations/jenkins/blueocean/detail/release%2F1.3/20/artifacts
(successful build) 

for `blueocean-dashboard.hpi` - it is almost 1MB smaller. 

There is a bug in the running of the bundling that causes it to ignore failures (in the log there will be some kind of lint failure no doubt.. which then causes the bundle to be skiipped). 

@cliffmeyers or @kzantow may remember the details, but this was fixed on master recently, but must bite here in some cases. 

So basically there is some small typo/validation error in the js, which tripps it, up, but I can't remember what to grep for...

**Comment by michaelneale** [comment]
trying a fix for this... it is tragic that I can spot these failure modes now ;)

**Comment by vivek** [comment]
@michaelneale thanks.
